packages/commonjs/test/fixtures/function/__esModule/main.js

```diff
@@ -2,4 +2,3 @@ import * as x from './answer';
 
 t.truthy('answer' in x);
 t.truthy('default' in x);
-t.truthy(!('__esModule' in x));
```

Comment:
With this change, `__esModule` is available as a named import. In my opinion this is correct, since it's exported from the commonjs module. What do you think?

Reply:
@lukastaegert this is something you'd have to approve. not speaking for the other maintainers, but I lack the knowledge to know if this is a good change.

Reply:
Usually the list of exports is detected via `Object.keys` which would skip the `__esModule` property since it is defined to be non-enumerable. I would be interested in how this list is being gathered that it is including it.